pkg/xds/conv/convertxds_test.go

```diff
@@ -649,22 +727,28 @@ func Test_convertStreamFilter_IsitoFault(t *testing.T) {
 }
 
 func Test_convertPerRouteConfig(t *testing.T) {
-	mixerFilterConfig := &client.ServiceConfig{
-		DisableReportCalls: false,
-		DisableCheckCalls:  true,
-		MixerAttributes: &v1.Attributes{
-			Attributes: map[string]*v1.Attributes_AttributeValue{
-				"test": &v1.Attributes_AttributeValue{
-					Value: &v1.Attributes_AttributeValue_StringValue{
-						StringValue: "test_value",
-					},
-				},
-			},
+	// mixerFilterConfig := &client.ServiceConfig{
+	// 	DisableReportCalls: false,
+	// 	DisableCheckCalls:  true,
+	// 	MixerAttributes: &v1.Attributes{
+	// 		Attributes: map[string]*v1.Attributes_AttributeValue{
+	// 			"test": &v1.Attributes_AttributeValue{
+	// 				Value: &v1.Attributes_AttributeValue_StringValue{
+	// 					StringValue: "test_value",
+	// 				},
+	// 			},
+	// 		},
+	// 	},
+	// }
+	mixerFilterConfig := client.HttpClientConfig{
+		Transport: &client.TransportConfig{
+			DisableReportBatch: false,
+			DisableCheckCache:  true,
 		},
 	}
 
-	mixerStruct := messageToAny(t, mixerFilterConfig)
-	fixedDelay := duration.Duration{}
+	mixerStruct := messageToAny(t, &mixerFilterConfig)
```

Comment:
What's the reason to modify this?

Reply:
i'm wrong rebuild mixerFilterConfig with client.HttpClientConfig